src/types/hull-traits-changes.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import { HullTraitNameType, HullTraitValueType } from "./";
+
+/**
+ * Trait changes is an object map where keys are trait names and value is an array
+ * where first element is an old value and second element is new value.
+ */
+export type HullTraitsChangesType = { [HullTraitNameType]: [HullTraitValueType, HullTraitValueType] };
```

Comment:
Same reason as above, I would suggest rename it to `HullAttributeChangesType` and since changes is already plural, i would suggest having attribute as singular.